app/resources/grammar.ebnf

```diff
@@ -0,0 +1,73 @@
+<program> ::= { <function> | <let_statement> | <comment> | <function_call> }
```

Comment:
What is this class for, isn't it the same as the AST we have?

Reply:
Actually it's, I decided to put it here for the ease of implementation of the parsers we have in the AST.

Reply:
It's not really necessary, but it's nice to have it in the repository.

Reply:
Yeah, I added during the process of the writing the parser for pattern matching because I need clear reference of the grammar.